src/platform/forms/fields.js

```diff
@@ -1,5 +1,4 @@
-import _ from 'lodash';
-
+import { keys, has, isPlainObject, mapValues, isArray } from 'lodash';
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11673/files#r377317714